commands/bot.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var BotCmd = &cobra.Command{
+	Use:   "bot",
+	Short: "Management of bots",
+}
+
+var CreateBotCmd = &cobra.Command{
+	Use:     "create [username]",
+	Short:   "Create bot",
+	Long:    "Create bot.",
+	Example: `  bot create testbot`,
+	RunE:    withClient(botCreateCmdF),
+	Args:    cobra.ExactArgs(1),
+}
+
+func init() {
+	CreateBotCmd.Flags().String("display-name", "", "Optional. The display name for the new bot.")
+	CreateBotCmd.Flags().String("description", "", "Optional. The description text for the new bot.")
+
+	BotCmd.AddCommand(
+		CreateBotCmd,
+	)
+
+	RootCmd.AddCommand(BotCmd)
+}
+
+func botCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	username := args[0]
+	displayName, _ := cmd.Flags().GetString("display-name")
```

Comment:
I think I had commented in an earlier review too. We should not be ignoring the err check here. I know it's like this everywhere. But if someone sets display-name as 5, then it will silently create the bot with an empty display name. It does not seem like the right behavior.

Feel free to do a sweep of this separately in a different PR though if you prefer.

Reply:
Generally it is done for getting the default value, but in this case it might be a good idea to check it. Good catch 👍

Reply:
I don't think this error checking is needed, tho we might decide to include it as it's a good practice. I believe that the check on the `Get*` methods is redundant, as the flag input types are checked as well at parse time. The only errors those could capture if I'm not mistaken are those for flags which values we set manually, or maybe I'm missing something?

In the case of setting `--display-name 5`, `5` is a valid string, so the resulting bot would be created with `displayName: 5` and that would be the expected behaviour, right?

Reply:
Yeah .. in case of GetString it makes sense. But what about GetBool or GetInt ? If they are verified at parse time, then I'm fine with it. The API shouldn't even have an error return then. But looking at the viper code, it does seem to do a type casting.

Reply:
My only explanation for that error return is just for cases where you set the flag manually, or those where you create (say) a `String` flag but call `GetInt` on that flag name. I tried to break the CLI with `Int` flags and wasn't able to, and for `Bool` flags they don't receive an arg (presence is the value) so I think we're fine as for safety.

In any case, we could include the checks for the shake of good practises if we think it's valuable, I'm not opposed to that.